.travis.yml

```diff
@@ -33,13 +33,6 @@ matrix:
   include:
     # Standard ORCA jobs.
     - { name: "Static code analysis", env: ORCA_JOB=STATIC_CODE_ANALYSIS }
-    - { name: "Deprecated code scan", env: ORCA_JOB=DEPRECATED_CODE_SCAN }
```

Comment:
We can't run any of these because ORCA tries to add Drupal 8.8 dependencies that conflict with our support of Drupal 8.7. I think webflo/drupal-core-require-dev might be the main problem.